factory/config.go

```diff
@@ -37,14 +37,16 @@ const (
 )
 
 type Configuration struct {
-	UdmName         string            `yaml:"udmName,omitempty"`
-	Sbi             *Sbi              `yaml:"sbi,omitempty"`
-	ServiceNameList []string          `yaml:"serviceNameList,omitempty"`
-	NrfUri          string            `yaml:"nrfUri,omitempty"`
-	WebuiUri        string            `yaml:"webuiUri"`
-	Keys            *Keys             `yaml:"keys,omitempty"`
-	PlmnSupportList []models.PlmnId   `yaml:"plmnSupportList,omitempty"`
-	PlmnList        []PlmnSupportItem `yaml:"plmnList,omitempty"`
+	UdmName                  string            `yaml:"udmName,omitempty"`
+	Sbi                      *Sbi              `yaml:"sbi,omitempty"`
+	ServiceNameList          []string          `yaml:"serviceNameList,omitempty"`
```

Comment:
This parameter seems to have a different name in other NFs (e.g., `ServiceList` in `pcf`). Should this be a good opportunity to align the parameters' name across the different NFs?

Reply:
This is replaced with ServiceList.